shared/team-building/team-box.tsx

```diff
@@ -28,10 +29,12 @@ const formatNameForUserBubble = (u: SelectedUser) => {
   switch (u.service) {
     case 'keybase':
     case 'contact': // do not display "michal@keyba.se on email" or similar
-    case 'phone':
     case 'email':
       technicalName = u.username
       break
+    case 'phone':
+      technicalName = formatPhoneNumber(u.username)
+      break
```

Comment:
The variable name `technicalName` reads like it's making a statement but I'm not sure what. Could be that it goes to the service, but it doesn't seem to. I suggest renaming it to something with less import like `name` or `displayName`